en/04_Changelogs/5.2.0.md

```diff
@@ -174,6 +175,18 @@ A `SearchableDropdownField` will now be used when automatically scaffolding `has
 
 Previously the [`DBForeignKey.dropdown_field_threshold`](api:SilverStripe\ORM\FieldType\DBForeignKey->dropdown_field_threshold) config property was used as the threshold of the number of options to decide when to switch between auto-scaffolding a `DropdownField` and a `NumericField`. This configuration property is now used as the threshold of the number of options to decide when to start using lazy-loading for the `SearchableDropdownField`.
 
+### Create file variants with different extensions {#file-variants}
+
+A low-level API has been added which allows the creation of file variants with a different extension than the original extension. A file variant is a manipulated version of the original file - for example if you resize an image or convert a file to another format, this will generate a variant (leaving the original file intact).
+
+Some examples of when you might want this are:
+
+- Generating thumbnails for videos, documents, etc
+- Converting images to to `.webp` for faster page load times
```

Comment:
```suggestion
- Generating thumbnails for videos, documents, etc.
- Converting images to `.webp` for faster page load times
```

Reply:
Removed the extra "to" - won't add the fullstop because we're not adding it to the other list items.